docs/source/webhooks.rst

```diff
@@ -172,6 +172,43 @@ To list all registered webhooks, run:
 
     st2 webhook list
 
+My Webhook Isn't Working!
+-------------------------
+
+Often you may run into issue with getting StackStorm to respond to webhooks the way you want. A common problem
+that community members run into is that StackStorm doesn't seem to recognize incoming webhooks, even if a manual
+test using ``curl`` results in a successful HTTP status.
+
+The first thing to do is confirm that the webhook request is indeed getting to StackStorm. When webhooks successfully
```

Comment:
Where possible, use |st2| rather than "StackStorm" - we have some macros for this.